docs/tutorial/using-checkbox/basic-launcher.rst

```diff
@@ -195,6 +195,96 @@ In Checkbox language, submissions files are the HTML test report as well as
 an archive containing the test results and additional logs that might have
 been produced by the test cases.
 
+A note about config files
+=========================
+
+So far, you have customized Checkbox using a launcher file. It is also
+possible to put these options in a configuration file that Checkbox will use
+when it is launched. The main difference is that you don't have to specify
+the launcher when running Checkbox.
+
+Create the file ``~/.config/checkbox.conf`` and add the following content
+in it:
+
+.. code-block:: none
+
+    [launcher]
+    launcher_version = 1
+    app_id = com.canonical.certification:tutorial
+    stock_reports = text, submission_files
+
+    [test plan]
+    unit = com.canonical.certification::TODO
+    forced = yes
+
+    [test selection]
+    forced = yes
+
+    [environment]
+    TUTO = tutorial
+
+Now, run Checkbox without any argument:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli
+
+You should see that Checkbox behaves exactly the same as in the previous
+section. It found the configuration from the ``~/.config/checkbox.conf``
+file and used it to automatically select the test plan and run it.
+
+Configuration files can be placed elsewhere on the system, and Checkbox
+will follow a certain resolution order to decide what configuration to
+use if more than one configuration files define the same key. Please check
+:ref:`checkbox_configs` for more information.
+
+Checkbox comes with a handy command to check what configuration is being used,
+and where it comes from. Run the following command:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli check-config
+    Configuration files:
+     - /var/snap/checkbox/2799/checkbox.conf
+     - /home/pieq/.config/checkbox.conf
```

Comment:
```suggestion
     - /home/user/.config/checkbox.conf
```
you might want to hide the real username, but it ok to keep if you don't mind :) (I see it is used in other files as well)